src/routes/Settings/modules/settings.js

```diff
@@ -36,6 +46,11 @@ const settings = (
     case CHANGE_PAYMENT_PASSWORD:
       console.log(action, state)
```

Comment:
fixme

Reply:
That's not mine. :) Will be fixed in Anton's PR. Didn't touch it to prevent merge conflicts

Reply:
Fixed in merge